app/utils/SessionKeys.scala

```diff
@@ -22,7 +22,7 @@ object SessionKeys {
   val endDateOfPeriod: String = "periodEnd"
   val dueDateOfPeriod: String = "periodDueDate"
   val dateCommunicationSent: String = "dateCommunicationSent"
-  val penaltyId: String = "penaltyId"
+  val penaltyNumber: String = "penaltyIdNumber"
```

Comment:
Could you change this to penaltyNumber?

Reply:
What do you want changed as this is already penaltyNumber?

Reply:
The value for 'penaltyNumber' is `penaltyIdNumber` can this be changed to `penaltyNumber`

Reply:
Good spot 👍 
Fixed ✅